docs/guides/getting-started-orchestrate-microservices.md

```diff
@@ -18,7 +18,9 @@ import SmPrereqs from './react-components/\_sm-prerequisites.md'
 import SaasPrereqs from './react-components/\_saas-prerequisites.md'
 import Install from './react-components/\_install-plain-java.md'
 
-Using Camunda 8, you can orchestrate the microservices necessary to achieve your end-to-end automated business process. Whether you have existing microservices or are looking to build out your microservices, this guide will help you understand how you can start your microservice orchestration journey with Camunda 8.
+Using Camunda 8, you can orchestrate the microservices necessary to achieve your end-to-end automated business process.
```

Comment:
If I am not mistaken, then this guide should have the "SaaS only" badge. Can you please check?